dokka-subprojects/analysis-kotlin-descriptors-compiler/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/descriptors/compiler/translator/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -840,15 +840,18 @@ private class DokkaDescriptorVisitor(
     private suspend fun visitTypeAliasDescriptor(descriptor: TypeAliasDescriptor) =
         with(descriptor) {
             coroutineScope {
+                val defaultType = runCatching { defaultType }.getOrNull() // `defaultType` can throw an exception for a recursive typealias A = A
```

Comment:
I think it's worth linking to #3565 for more details, and it'll be easier to find this place in the future if needed

```suggestion
                // `defaultType` can throw an exception for a recursive typealias A = A
                // for more details see https://github.com/Kotlin/dokka/issues/3565
                val defaultType = runCatching { defaultType }.getOrNull()
```